### ISSUE TITLE
Lovelace UI support?

### ISSUE THREAD

**Issue by andreipopovici** [andrey-git/home-assistant-custom-ui#139]
Hi there,

Are there any plans (or ideas) for integrating Custom UI elements (in my case specifically, inline sliders for lights) with the new, experimental Lovelace UI?

The [Lovelace Custom Cards docs](https://developers.home-assistant.io/docs/en/lovelace_custom_card.html) seem a little different than how Custom UI works currently.

Thank you!

---

### Checklist
- [x] I'm running the latest version of CustomUI ([Update guide](https://github.com/andrey-git/home-assistant-custom-ui/blob/master/docs/installing.md#updating)) or using a specific [release](https://github.com/andrey-git/home-assistant-custom-ui/releases) that is not marked as "Broken".
- [x] I tried to force-refresh (Ctrl+Shift+R / Ctrl+F5) the browser
- [x] (Optional, but recommended) I'm using Chrome or tried to reproduce the feature on Chrome.

**Browser + Version:** 

**CustomUI version:** 

**Home Assistant release (`hass --version`):** 
0.72.1

**Problem-relevant `configuration.yaml` entries:**
```yaml

```

**Problem-relevant Home Assistant log entries:**
```

```

**Any errors from browser Javascript console:**
```

```

**Comment by andrey-git** [comment]
I'll get to it when Lovelace is more stable.

**Comment by isabellaalstrom** [comment]
@andrey-git Do you need some help with that? I don't know if it's still regarded as unstable since this weekends release? I really want to use Lovelace but I really feel I can't since I love your addition to the ui too much. Can't live without light sliders! 😥

**Comment by andreipopovici** [comment]
That's the main feature of @andrey-git's Custom UI that I'm using, too. However, I think with the way Lovelace is designed, it would either have to be a custom card, or it'd have to be integrated directly into Lovelace's codebase.

**Comment by VirtualL** [comment]
Any news? I Really Love the Custom-UI and its very much missing in the new Lovelace UI

**Comment by andrey-git** [comment]
It will require significant amount of work. I started working on it, but I spend significantly less time on HA now.

**Comment by tringler** [comment]
@andrey-git Do you know how long home-assistant will support the old frontend? My whole configuration is full and depended of custom-ui :(

**Comment by iantrich** [comment]
@tringler the default will be switched to Lovelace in the next few releases, I believe, but the old UI is not going away that soon.

**Comment by gcorgnet** [comment]
Just curious to see if there's been any progress towards this. It still seems like the light slider will not show up in lovelace.

**Comment by iantrich** [comment]
@gcorgnet not sure why you say that? You want light slider in lovelace, build a light slider for lovelace and submit it? Or just search for a custom one https://github.com/thomasloven/lovelace-slider-entity-row

**Comment by horacecar** [comment]
Yes!!!!! most of my feature is also from Custom-ui specially the lock feature..!! super like! but i designed my card with lovelace, then the lock feature and most of Custom-ui feature cannot use with lovelace , only states UI can use , that's too regrettably....

**Comment by VirtualL** [comment]
@horacecar Give us some info, what are you planning to do :)
